pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
I don't think there is any reason you'd need to retain arbitrary precision in wAIC values. Differences less than ~2 correspond to equivalent models.